lang/cs/Org.Apache.REEF.Common/metrics/Api/IMetricsSink.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+
+namespace Org.Apache.REEF.Common.Metrics.Api
+{
+    /// <summary>
+    /// The metrics sink interface. Via this interface user/application 
+    /// consumes and access the metrics. <see cref="IMetricsSystem"/> will periodically 
+    /// push records from source to sink using the OnNext method of the interface. 
+    /// </summary>
+    public interface IMetricsSink : IDisposable, IObserver<IMetricsRecord>
```

Comment:
We can ..... I just thought of explicitly calling it out so that people understand that there is a sink. I will remove it.